test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/survival/KieServerWithWorkbenchSurvivalIntegrationTest.java

```diff
@@ -0,0 +1,164 @@
+        workbenchWithKieServerScenario.deploy();
+
+        gitProvider = GitProviderFactory.getGitProvider();
+        gitProvider.createGitRepository(workbenchWithKieServerScenario.getNamespace(), ClassLoader.class.getResource("/kjars-sources").getFile());
+
+        workbenchClientProvider = new WorkbenchClientProvider(workbenchWithKieServerScenario.getWorkbenchDeployment());
+        workbenchClientProvider.createOrganizationalUnit(ORGANIZATION_UNIT_NAME, workbenchWithKieServerScenario.getWorkbenchDeployment().getUsername());
+        workbenchClientProvider.cloneRepository(ORGANIZATION_UNIT_NAME, REPOSITORY_NAME, gitProvider.getRepositoryUrl(workbenchWithKieServerScenario.getNamespace()));
+        workbenchClientProvider.deployProject(REPOSITORY_NAME, PROJECT_NAME);
+
+        kieServerClientProvider = new KieServerClientProvider(workbenchWithKieServerScenario.getKieServerDeployment());
+
+        kieServerControllerClientProvider = new KieServerControllerClientProvider(workbenchWithKieServerScenario.getWorkbenchDeployment());
+        kieServerMgmtControllerClient = kieServerControllerClientProvider.getKieServerMgmtControllerClient();
+        kieServerControllerClientProvider.waitForServerTemplateCreation();
+    }
+
+    @After
+    public void tearDown() {
+        InstanceLogUtil.writeDeploymentLogs(workbenchWithKieServerScenario.getWorkbenchDeployment());
+        InstanceLogUtil.writeDeploymentLogs(workbenchWithKieServerScenario.getKieServerDeployment());
+        workbenchWithKieServerScenario.undeploy();
+        gitProvider.deleteGitRepository(workbenchWithKieServerScenario.getNamespace());
+    }
+
+    @Test
+    public void kieServerScaleTest() {
+        KieServerInfo serverInfo = kieServerClientProvider.getKieServerClient().getServerInfo().getResult();
+        checkServerTemplateIsRegistred(serverInfo.getServerId(), serverInfo.getName());
+
+        scaleKieServerTo(0, serverInfo.getServerId());
+
+        scaleKieServerTo(1, serverInfo.getServerId());
+
+        logger.debug("Register Kie Container to Kie Server");
+        kieServerMgmtControllerClient.saveContainerSpec(serverInfo.getServerId(), serverInfo.getName(), CONTAINER_ID, CONTAINER_ALIAS, PROJECT_GROUP_ID, PROJECT_NAME, PROJECT_VERSION, KieContainerStatus.STARTED);
+        kieServerClientProvider.waitForContainerStart(CONTAINER_ID);
+
+        logger.debug("Start process instance");
+        Long signalPid = kieServerClientProvider.getProcessClient().startProcess(CONTAINER_ID, SIGNALTASK_PROCESS_ID);
+        Assertions.assertThat(signalPid).isNotNull().isGreaterThan(0L);
+        Assertions.assertThat(kieServerClientProvider.getProcessClient().findProcessInstances(CONTAINER_ID, 0, 10)).isNotNull().hasSize(1);
+
+        scaleKieServerTo(0, serverInfo.getServerId());
+
+        logger.debug("Start process instance");
+        try {
+            Long noContainerPid = kieServerClientProvider.getProcessClient().startProcess(CONTAINER_ID, SIGNALTASK_PROCESS_ID);
```

Comment:
This try/catch approach isn't ideal. It is not clear what should be the result and what do you verify in that case. Try to use Assertions.assertThatThrownBy() instead.